tests/unit/ds-macro-test.js

```diff
@@ -37,3 +37,32 @@ test('DS macro', function(assert) {
 
   assert.deepEqual(relationships, ['user', 'car']);
 });
+
+class ES6Model extends DS.Model {
```

Comment:
that's what I had initially, but my tests kept failing to start. Not sure how to best reset my environment.

Errors I've gotten:
Cannot find module '../Doc/FileDoc.js'
TypeError: ora is not a function